src/components/visualizations/chart/highcharts/dataLabelsHelpers.ts

```diff
@@ -149,3 +156,41 @@ export function getShapeVisiblePart(shape: any, chart: any, wholeSize: number) {
 
     return wholeSize;
 }
+
+export function getLabelStyle(type: string, stacking: string) {
+    if (isAreaChart(type)) {
+        return BLACK_LABEL;
+    }
+    return stacking || isOneOfTypes(type, whiteDataLabelTypes) ? WHITE_LABEL : BLACK_LABEL;
+}
+
+export function formatAsPercent(unit: number = 100): string {
+    const val = parseFloat((this.value * unit).toPrecision(14));
```

Comment:
I know you just moved the existing code but please `this.` here!!! not inside class? 

please create a ticket for this to fix this function without using "this"

also this place: https://github.com/gooddata/gooddata-react-components/blob/481391606d649919c65af06383cdd7d623e192d3/src/components/visualizations/chart/highcharts/plugins/dualAxesLabelFormatter.ts#L122 
without using `this` please

Reply:
Reported in https://jira.intgdc.com/browse/SD-484